core/src/main/java/org/vertexium/type/GeoPoint.java

```diff
@@ -109,7 +109,7 @@ public boolean intersects(GeoShape geoShape) {
 
     @Override
     public boolean within(GeoShape geoShape) {
-        throw new VertexiumException("Not implemented for argument type " + geoShape.getClass().getName());
+        return false;
```

Comment:
Same here a point can't be inside a geo poly?

Reply:
Same as above. It feels backwards, but based on the `GeoRect` implementation this appears to be asking if the `GeoShape` is within the point which is always false.

Reply:
Done